script/Arcueid/Day 1/01_00_ARC01_1_2.txt

```diff
@@ -131,23 +131,22 @@ Her smile is so friendly that I can't help but joke a little.
 [sha:ab01ead12face23ea9d4c3b98e388317f55e6f2f]{
 -- Page 56, Offset 1291.
 -- 緑色のリボンの先輩は、
-The upperclassman with a green ribbon replies―――
+The upperclassman with the green ribbon replies―――
 }
 [sha:8c0ad9597e7875db75fe7655ab0ed5b530c6b8e9]{
 -- Page 56, Offset 1292.
 -- 「どちらかというとワンちゃんではないでしょうか。
-"It's more like a dog, I think.
+"It's more like a puppy, I think.
 }
 [sha:2f290989076c056ff31e11aad499df4f8ee0e870]{
 -- Page 56, Offset 1293.
 -- というかネコ科はよくありません。最後まで頼りになるのは、律儀で頭のいいイヌ科です！」
-I mean, felines are no good. The ones you can rely on to the end are the disciplined and intelligent canines!"
+I mean, felines are awful. Dutiful and intelligent canines are the ones you can truly count on in the end!"
 }
 [sha:ae9a64034cf4a7a9f1e05cd48b49a44dcee49cda]{
 -- Page 56, Offset 1294.
 -- 得意そうに、よく分からないこだわりを見せるのだった。
-She looks pleased with herself,
- I feel like I've been shown a glimpse of a hitherto unknown obsession.
+She lays out her oddly strong preferences with a pleased look on her face.
```

Comment:
She has really shoehorned this into the conversation so maybe:
She declares her oddly strong preferences with a pleased look on her face.

or

She seems really pleased to declare her oddly definitive preferences. I didn't realise she felt strongly about this very specific topic.

Reply:
former sounds good to me.